Jenkinsfile

```diff
@@ -31,13 +31,19 @@ pipeline {
 
     stage("Build") {
       steps {
-        sh "npm --unsafe-perm install"
-        sh "npm run build"
```

Comment:
We still need to keep linting on the CI + switching to `build-assets` will omit `prebuild` stage.

```
"build-assets": "npm run clean && lingui compile && NODE_ENV=production webpack --colors --config ./webpack/webpack.production.js",
./dist/index.html {} \\;",
"prebuild": "./scripts/validate-tests",
"build": "npm run lint && ([ -z \"${npm_config_externalplugins}\" ] || npm run lint-plugins) && npm test && npm run build-assets && npm run validate-build",
```

Reply:
I see, but if I'd move prebuild to pretest this should be fine, shouldn't it?

Reply:
Yup, thats better anyway

Reply:
I don't see the comment being resolved

Reply:
See the package.json

Reply:
We still skipping  `validate-build` step. That could mean once we merge master is broken. 
Shall we move it to `postbuild-assets?`